Sources/ScipioKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -220,6 +256,9 @@ class ProjectGenerator {
             ])
         )
 
+        // Make Resource target if needed
```

Comment:
```suggestion
        // Make Resource target if needed (returns nil if not necessary)
```

we may expect this throw error if not needed, please add comment